distribution/bwc/build.gradle

```diff
@@ -148,6 +153,8 @@ if (enabled) {
     }
   }
 
+  // ensure build metadata is written, even if nothing uses it in this build
+  check.dependsOn writeBuildMetadata
```

Comment:
I wonder about this, is it enough? Could we write out a commit for `bwc-release-snapshot` that we are not necessarily compatible with since it's not actually tested in this build?

Reply:
You are right, we could. The only other thing to do is add this as an additional version for the full restart qa test when run with `gradle check`.  That would at least guarantee we had basic compatibility (but it does mean the intake job will drag on a bit longer).  I'm ok with that though. I will update shortly.